src/Rules/Library/BoundingRectangleNotNullListViewXAML.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Axe.Windows.Rules.PropertyConditions;
+using Axe.Windows.Rules.Resources;
+using static Axe.Windows.Rules.PropertyConditions.BoolProperties;
+using static Axe.Windows.Rules.PropertyConditions.ElementGroups;
+
+namespace Axe.Windows.Rules.Library
+{
+    [RuleInfo(ID = RuleId.BoundingRectangleNotNullListViewXAML)]
+    class BoundingRectangleNotNullListViewXAML : Rule
+    {
+        public BoundingRectangleNotNullListViewXAML()
+        {
+            this.Info.Standard = A11yCriteriaId.ObjectInformation;
+            this.Info.PropertyID = Axe.Windows.Core.Types.PropertyType.UIA_BoundingRectanglePropertyId;
+            this.Info.Description = Descriptions.BoundingRectangleNotNull;
```

Comment:
Curiosity question: I see that this uses the same description as `BoundingRectangleNotNull`, is this expected?

Reply:
Yes. The description is intended to explain what the symptom of the problem is, and since we're still checking the same thing in different frameworks, the same description applies. We've followed this pattern in other cases where we created more targeted versions of an existing rule. It has a downside, in that it violates the concept of a 1:1 relationship between description and rule. As a result, if we receive an issue containing just the description, we also need to look at the framework information to determine which rule is in use. In actual practice, we'd have to do that anyway, so it's not a significant penalty.

Reply:
Makes sense, thank you!